sparrow-py/pysrc/sparrow_py/_expr.py

```diff
@@ -345,7 +367,7 @@ def _aggregation(
     # us to add the *args like so.
     if window is None:
         return Expr.call(op, *args, input, None, None)
-    elif isinstance(window, SinceWindow):
+    elif isinstance(window, kt.SinceWindow):
```

Comment:
do you have to do `kt.` since you've already imported it
```
from ._windows import SinceWindow
```
if so, or not, should we do the same for `SlidingWindow` below?

Reply:
Dropped the import and switched to `kt`. There are some painful things about how imports within the same package are resolved (seems to lead to lots of circular imports). We may end up wanting to import specific modules (files) from within `sparrow_py` and only do the `import sparrow_py as kt` thing in tests / outside the package.

For instance, if you want `class Window(kt.Expr)` then that doesn't work if the window file is imported before the expr file in the `__init__.py` in `kt`, since when the window is imported, it hasn't yet been defined in the `__init__` module.

I think things are working now... but if it continues to be a problem I may suggest that within `pysrc` we always import from files (using relative paths).